notices/ltc-asset-transfer.vbs

```diff
@@ -56,49 +57,96 @@ changelog_display
-    CancelButton 250, 80, 50, 15
-EndDialog
+
+'Listing button numbers for links
+onesource_URL = 101
+epm_URL = 102
+button_fill = 103
 
 Do
     Do
-        err_msg = ""
-        Dialog Dialog1
-        cancel_without_confirmation
-        IF IsNumeric(maxis_case_number) = false or len(maxis_case_number) > 8 then err_msg = err_msg & vbNewLine & "* Enter a valid case number."
-        If trim(renewal_footer_month) = "" or len(renewal_footer_month) <> 2 then err_msg = err_msg & vbNewLine & "* Enter a two-digit renewal month."
-        If trim(renewal_footer_year) = "" or len(renewal_footer_year) <> 2 then err_msg = err_msg & vbNewLine & "* Enter a two-digit renewal year."
+      err_msg = ""
+      fill_err_msg = ""
+
+      Dialog1 = ""    
+      BeginDialog Dialog1, 0, 0, 301, 140, "LTC asset transfer dialog"
+        EditBox 75, 5, 50, 15, MAXIS_case_number
+        ButtonGroup ButtonPressed
+          PushButton 135, 5, 160, 15, "Press HERE to autofill fields below.", button_fill
+        EditBox 75, 30, 20, 15, renewal_footer_month
+        EditBox 100, 30, 20, 15, renewal_footer_year
+        EditBox 75, 55, 70, 15, client
+        EditBox 225, 55, 70, 15, spouse
+        EditBox 75, 80, 220, 15, worker_signature
+        ButtonGroup ButtonPressed
+          OkButton 185, 120, 50, 15
+          CancelButton 245, 120, 50, 15
+          PushButton 20, 115, 40, 15, "OneSource", onesource_URL
+          PushButton 70, 115, 40, 15, "EPM", epm_URL
+        Text 30, 10, 45, 10, "Case number:"
+        Text 15, 35, 60, 10, "ER date (MM/YY):"
+        Text 5, 60, 70, 10, "  Resident First Name:"
+        Text 160, 60, 65, 10, "Spouse First Name:"
+        Text 15, 85, 60, 10, " Worker Signature:"
+        GroupBox 15, 105, 100, 30, "LTC Asset Transfer Policies"
+      EndDialog
+
+      Dialog Dialog1
+      cancel_without_confirmation
+  
+      If ButtonPressed > 100 Then
+        err_msg = "Loop"
+
+        If ButtonPressed = onesource_URL Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe https://www.dhs.state.mn.us/main/idcplg?IdcService=GET_DYNAMIC_CONVERSION&RevisionSelectionMethod=LatestReleased&dDocName=ONESOURCE-170126"
+        If ButtonPressed = epm_URL Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe http://hcopub.dhs.state.mn.us/epm/2_4_2_1_2.htm?rhhlterm=12%20community%20spouse&rhsearch=12%20months%20community%20spouse"
+        If ButtonPressed = button_fill Then 
+          Call Check_for_MAXIS(false)                         'Ensuring we are not passworded out  
+          Call validate_MAXIS_case_number(fill_err_msg, "*")
+          If fill_err_msg <> "" Then 
+            MsgBox "Enter the required information specified below before selecting autofill:" & fill_err_msg
+          Else
+            Back_to_SELF
+            Call navigate_to_MAXIS_screen("STAT","REVW")
+            CALL write_value_and_transmit("X", 5, 71)
+            EMReadScreen renewal_footer_month, 2, 9, 27
```

Comment:
This will autofill '__' for the footer months and years if the case does not have a HC ER coded on REVW.